packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -225,52 +225,44 @@ export class GarbageCollector implements IGarbageCollector {
 			this.getNodePackagePath,
 		);
 
-		// Get the GC data from the base snapshot. Use LazyPromise because we only want to do this once since it
-		// it involves fetching blobs from storage which is expensive.
-		this.baseSnapshotDataP = new LazyPromise<IGarbageCollectionSnapshotData | undefined>(
-			async () => {
-				if (baseSnapshot === undefined) {
+		// Get the GC data from the base snapshot. Ensure we only do this once since it
+		// involves fetching blobs from storage which is expensive.
+		this.getBaseSnapshotData = onceifyFunction(async () => {
+			if (baseSnapshot === undefined) {
```

Comment:
There are no actual changes to this callback function, prettier just decided to change all the indentation